src/components/tech/HeroSection.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import { Box, Flex, Heading, Text, Button } from "@radix-ui/themes";
+import heroBackgroundImg from "../../../public/images/tech/tech-hero-background-img.jpg";
+const HeroSection = () => {
+  return (
+    <div className="block w-[1252px] item-center justify-center p-8 relative ">
+      <h1 className="text-center text-white uppercase font-bold md:text-5xl md:py-6 py-2 z-20 sm:text-3xl text-xl">
```

Comment:
**suggestion (non-blocking):** Use RadixUI's [Heading Component](https://www.radix-ui.com/themes/docs/components/heading) if possible.

**suggestion (non-blocking):** Wrap with RadixUI's [Block Component](https://www.radix-ui.com/themes/docs/components/box) with the `asChild` property set in order to pass margin / padding props to the heading.

Reply:
@tahirxx my bad, it should be `Box`